charts/generic-service/templates/pdb.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if .Values.poddisruptionbudget.enabled }}
-apiVersion: policy/v1beta1
+apiVersion: policy/v1
```

Comment:
Because we use
```
spec:
  minAvailable: {{ .Values.poddisruptionbudget.minAvailable }}
  selector:
    matchLabels:
      app: {{ include "app.name" . }}
```
I don't think we need to make any changes

See [CloudPlatform guide](https://user-guide.cloud-platform.service.justice.gov.uk/documentation/other-topics/apiversion-changes-pdb-k8s-1-25.html#steps-required) for differences between `policy/v1beta1` and `policy/v1`